src/extensions/default/CSSPseudoSelectorHints/PseudoSelectors.json

```diff
@@ -1,30 +1,36 @@
     {
         "active":              {"desc": "Selects the active link"},
         "checked":             {"desc": "Selects every checked <input> element"},
+        "default":             {"desc": "Selects every UI element that is the default among a group of similar elements"},
+        "dir(direction)":      {"desc": "Selects every element whose text direction is 'direction'", "text": "dir()"},
         "disabled":            {"desc": "Selects every disabled <input> element"},
         "empty":               {"desc": "Selects every element that has no children/text (including text nodes)"},
         "enabled":             {"desc": "Selects every enabled <input> element"},
         "first-child":         {"desc": "Selects every element that is the first child of its parent"},
         "first-of-type":       {"desc": "Selects every element that is the first element identified by 'type' of its parent"},
         "focus":               {"desc": "Selects the input element which has focus"},
+        "focus-within":        {"desc": "Selects every element which or whose descendant has focus"},
         "hover":               {"desc": "Selects elements on mouse over"},
         "in-range":            {"desc": "Selects input elements with a value within a specified range"},
+        "indeterminate":       {"desc": "Selects every indeterminate checkbox or radio button"},
         "invalid":             {"desc": "Selects all input elements with an invalid value"},
-        "lang(language)":      {"desc": "Selects every element with a lang attribute equal to 'language'", "text":"lang()"},
+        "lang(language)":      {"desc": "Selects every element with a lang attribute equal to 'language'", "text": "lang()"},
         "last-child":          {"desc": "Selects every element that is the last child of its parent"},
         "last-of-type":        {"desc": "Selects every element that is the last element of its parent"},
         "link":                {"desc": "Selects all unvisited links"},
-        "not(selector)":       {"desc": "Selects every element that is not an element identified by 'selector'", "text":"not()"},
-        "nth-child(n)":        {"desc": "Selects every element that is the second child of its parent", "text":"nth-child()"},
-        "nth-last-child(n)":   {"desc": "Selects every element that is the second child of its parent, counting from the last child", "text":"nth-last-child()"},
-        "nth-last-of-type(n)": {"desc": "Selects every element that is the nth element of its parent, counting from the last child", "text":"nth-last-of-type()"},
-        "nth-of-type(n)":      {"desc": "Selects every element that is the nth element of its parent",  "text":"nth-of-type(n)"},
-        "only-of-type":        {"desc": "Selects every element that is the only element of this type of its parent"},
+        "matches(selectors)":  {"decs": "Selects every element that is matched by one or more selectors in the 'selectors' list"},
```

Comment:
Also add a 'text' field in the value with the insertion text as 'matches()'. Other wise user has to always delete selectors after hint selection.